### PR TITLE
Modify log_in method in SessionsHelper

### PR DESCRIPTION
**Pull request by ghost**
-Modified the log_in method in the SessionsHelper: now it automatically sets the current_user as the logged in user.
-Added rspec for helpers/sessions_helper.rb
-Added SessionHelper to rails_helper.rb for "helper" types.

### CODE REVIEW

File: spec/helpers/sessions_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe SessionsHelper, type: :helper do
+  let(:user) do
+    User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+  end
+
+  describe '#log_in' do
+    subject { log_in(user) }
+    let(:logged_in_user) { user.id }
```
Comment (luukveenis) [inline spec/helpers/sessions_helper_spec.rb]: These `let`s make the test harder to read in my opinion, I would just:
`it { is_expected.to be user.id }`

File: spec/helpers/sessions_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    let(:logged_in_user) { user }
+    before { log_in(user) }
+    it { is_expected.to eq logged_in_user }
+  end
+
+  describe '#log_out' do
+    subject { log_out }
+    before do
+      log_in(user)
+      current_user
```
Comment (luukveenis) [inline spec/helpers/sessions_helper_spec.rb]: I'm not sure what `current_user` does, but does it need to be called here?

File: spec/helpers/sessions_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    end
+    it { expect { subject }.to change { session[:user_id] } }
```
Comment (luukveenis) [inline spec/helpers/sessions_helper_spec.rb]: This example is checking more than just a simple return value, so I would prefer a better description on the example, ie:
```
it "sets the current user id in the session" do
  ...
end
```

File: spec/helpers/sessions_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    it do
```
Comment (luukveenis) [inline spec/helpers/sessions_helper_spec.rb]: same here, a better description would be great

**Review by luukveenis — CHANGES_REQUESTED** [review]

**Comment by cbrunsdon** [comment]
jenkins test this please

File: spec/helpers/sessions_helper_spec.rb (hunk 1 above)
Comment (ghost) [reply spec/helpers/sessions_helper_spec.rb]: Thanks for all the feedback, Luuk! I hope I made it better this time.

**Review by qr8r — APPROVED** [review]

**Review by luukveenis — APPROVED** [review]

**Pull request merged**